Marlin/src/pins/stm32f4/pins_BTT_E3_RRF.h

```diff
@@ -0,0 +1,374 @@
+  #define E1_DIR_PIN                    FPC4_PIN  // E1DIR
+  #define E1_SERIAL_TX_PIN              FPC6_PIN  // E1UART
+  #define E1_SERIAL_RX_PIN              FPC6_PIN  // E1UART
+
+  #ifndef FIL1_RUNOUT2_PIN
+    #define FIL_RUNOUT2_PIN             FPC3_PIN  // E1-STOP
+  #endif
+
+  #define PT100_PIN                     FPC8_PIN  // Analog Input "PT100"(INA826)
+  #define TEMP_1_PIN                    FPC9_PIN  // Analog Input "TH1"
+
+  #define FAN1_PIN                     FPC15_PIN  // "FAN0" in IDEX board
+  #define FAN2_PIN                     FPC14_PIN  // "FAN1" in IDEX board
+
+#endif
+
+// Onboard I2C EEPROM
+#define I2C_EEPROM
+#define MARLIN_EEPROM_SIZE 0x1000                 // 4KB
+
+//
+// Servos
+//
+#define SERVO0_PIN                          PB0   // SERVOS
+
+//
+// Limit Switches
+//
+#define X_STOP_PIN                          PC0   // X-STOP
+#define Y_STOP_PIN                          PC1   // Y-STOP
+#define Z_STOP_PIN                          PC2   // Z-STOP
+
+//
+// Z Probe must be this pin
+//
+#define Z_MIN_PROBE_PIN                     PC5   // PROBE
+
+//
+// Filament Runout Sensor
+//
+#ifndef FIL_RUNOUT_PIN
+  #define FIL_RUNOUT_PIN                    PC3   // E0-STOP
+#endif
+
+//
+// Power-loss Detection
+//
+#ifndef POWER_LOSS_PIN
+  #define POWER_LOSS_PIN                    PE0   // Power Loss Detection: PWR-DET
+#endif
+
+//
+// Steppers
+//
+#define X_ENABLE_PIN                        PD7
+#define X_STEP_PIN                          PD5
+#define X_DIR_PIN                           PD4
+
+#define Y_ENABLE_PIN                        PD3
+#define Y_STEP_PIN                          PD0
+#define Y_DIR_PIN                           PA15
+
+#define Z_ENABLE_PIN                        PD14
+#define Z_STEP_PIN                          PC6
+#define Z_DIR_PIN                           PC7
+
+#define E0_ENABLE_PIN                       PD10
+#define E0_STEP_PIN                         PD12
+#define E0_DIR_PIN                          PD13
+
+/**
+ * TMC2208/TMC2209 stepper drivers
+ */
+#if HAS_TMC_UART
+  //
+  // Software serial
+  //
+  #define X_SERIAL_TX_PIN                   PD6
+  #define X_SERIAL_RX_PIN                   PD6
+
+  #define Y_SERIAL_TX_PIN                   PD1
+  #define Y_SERIAL_RX_PIN                   PD1
+
+  #define Z_SERIAL_TX_PIN                   PD15
+  #define Z_SERIAL_RX_PIN                   PD15
+
+  #define E0_SERIAL_TX_PIN                  PD11
+  #define E0_SERIAL_RX_PIN                  PD11
+
+  // Reduce baud rate to improve software serial reliability
+  #define TMC_BAUD_RATE 19200
+#endif
+
+//
+// Temperature Sensors
+//
+#define TEMP_BED_PIN                        PA1   // Analog Input "TB"
+#define TEMP_0_PIN                          PA0   // Analog Input "TH0"
+
+//
+// Heaters / Fans
+//
+#define HEATER_BED_PIN                      PB4   // "HB"
+#define HEATER_0_PIN                        PB3   // "HE0"
+#define HEATER_1_PIN                        PE4   // "HE1"
```

Comment:
This line should be moved to somewhere around line 73 (it  is on IDEX module) and should look like 
`#define HEATER_1_PIN                        FPC16_PIN   // "HE1"`